docs/architecture.md

```
@@ -74,7 +74,11 @@ src/
 
 | Project                   | Technologies                                                                                                           | Implements                                                                              |
 | ------------------------- | ---------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+<<<<<<< HEAD
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)